pkg/analytics/googleanalytics.go

```diff
@@ -0,0 +1,38 @@
+package analytics
```

Comment:
Add the license field on the Top, Check from the other file

Reply:
Done @chandankumar4